code/modules/mob/living/living.dm

```diff
@@ -838,6 +838,14 @@ Thanks.
 			qdel(package)
 			playsound(src.loc, 'sound/items/poster_ripped.ogg', 100, 1)
 		return
+	else if(istype(src.loc, /obj/effect/spider/cocoon))
+		var/obj/effect/spider/cocoon/cocoon = src.loc
+		to_chat(L, "<span class='warning'>You attempt to untangle yourself, the webs are tight and will take some time.</span>")
+		if(do_after(src, src, 2 MINUTES))
+			L.visible_message("<span class='danger'>[L] successfully breaks out of [cocoon]!</span>",\
+							  "<span class='notice'>You successfully break out!</span>")
+			forceMove(T)
```

Comment:
shouldn't this be L.forceMove?

Reply:
I'm just copying the other code that's already there and it works

Reply:
What are you trying to do with this line of code? It appears to move the cocoon to the tile that the cocoon is on.

Reply:
I couldn't tell you what the code does, I just know it works and I copied it. I really can't say more about it, sorry

Reply:
Well here `src` is the guy doing resist, so all that using `L` does is a typecheck.
It's probably fine.